packages/rea/rea.0.2.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Effectful OCaml with Objects and Variants"
+maintainer: "Vesa Karvonen <vesa.a.j.k@gmail.com>"
+authors: "Vesa Karvonen"
+license: "MIT"
+homepage: "https://github.com/polytypic/rea-ml"
+bug-reports: "https://github.com/polytypic/rea-ml/issues"
+dev-repo: "git+https://github.com/polytypic/rea-ml.git"
+depends: [
+  "dune" {>= "3.3.0"}
+  "ocaml" {>= "4.08.0"}
+  "ppx_optcomp" {>= "v0.14.0"}
+  "ocamlformat" {dev}
+  "odoc" {dev}
```

Comment:
These should not be dev dependencies. See https://github.com/ocaml/opam/issues/4959
```suggestion
```

Reply:
Thanks for the note!